### PR TITLE
Option to append access token to success redirect URL

### PR DESCRIPTION
**Pull request by jeff3yan**
Added an option for appending the access token as a query parameter on the success redirect URL. Similar to this [fork](https://github.com/asvdw/loopback-component-passport/commit/300579dd4b3157352b98eea831276cff890fb627), but with a config option `redirectWithToken`.

This was desired due to loopback-component-passport 3rd party login only supporting access token setting via cookies, which isn't feasible when the front-end and API are on different domains.

Similar in intent to #102, but it allows continued usage of the `returnTo` query parameter and is useful for apps that may have dynamic redirects on separate domains.

return res.redirect(successRedirect(req) + (options.redirectWithToken ? '?access_token=' + info.accessToken.id : ''));

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by jannyHou** [comment]
@slnode test please

**Comment by Casy** [comment]
please, make it

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

File: lib/passport-configurator.js [hunk 1]
```diff
@@ -463,16 +463,16 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
...
                 res.cookie('userId', user.id.toString(), {
                   signed: req.signedCookies ? true : false,
                   maxAge: 1000 * info.accessToken.ttl,
-                  domain: (options.domain) ? options.domain : null,
+                  domain:  typeof options.cookieDomain !== 'undefined' ? options.cookieDomain : (options.domain || null)
                 });
               }
             }
-            return res.redirect(successRedirect(req));
+            return res.redirect(successRedirect(req) + (options.redirectWithToken ? '?access_token=' + info.accessToken.id : ''));
```
Comment (jannyHou) [inline lib/passport-configurator.js]: I am not sure is it safe to expose accessToken in url, @loay idea?

**Comment by divsbhalala** [comment]
how can i use this feature ? any doc ?

**Pull request closed without merging**